talk/objectorientation/advancedoo.tex

```diff
@@ -491,9 +491,12 @@
   \frametitlecpp[98]{Virtual inheritance}
   \begin{block}{Solution}
     \begin{itemize}
-    \item inheritance can be {\it virtual} or not
-    \item {\it virtual} inheritance will ``share'' parents
-    \item standard inheritance will replicate them
+    \item inheritance can be \cppinline{virtual} or not
+    \begin{itemize}
+      \item \cppinline{virtual} inheritance will ``share'' parents
+      \item standard inheritance will replicate them
+    \end{itemize}
+    \item most derived class calls the virtual base class's constructor
```

Comment:
```suggestion
    \item most derived class will call the virtual base class's constructor
```

I misread this at first as "class-calls". By adding the "will", this confusion doesn't happen.